.github/workflows/release-please.yml

```diff
@@ -18,6 +18,32 @@ jobs:
           signoff: "OpenFeature Bot <109696520+openfeaturebot@users.noreply.github.com>"
     outputs:
       release_created: ${{ steps.release.outputs.releases_created }}
+      all: ${{ toJSON(steps.release.outputs) }}
+      paths_released: ${{ steps.release.outputs.paths_released }}
+
+  sbom:
+    needs: release-please
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        release: ${{ fromJSON(needs.release-please.outputs.paths_released) }}
+    env:
+      TAG: ${{ fromJSON(needs.release-please.outputs.all)[format('{0}--tag_name', matrix.release)] }}
+    steps:
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+      - name: Generate SBOM
+        run:
+          npm install -g npm@^10.2.0
```

Comment:
Why is this necessary? Does Node version 18 not ship with a new enough version of NPM?

Reply:
Thats what I am not sure about.
When I installed the newest node 18 via NVM I got: 

```
❯ nvm install 18
Downloading and installing node v18.18.2...
Downloading https://nodejs.org/dist/v18.18.2/node-v18.18.2-darwin-arm64.tar.xz...
######################################################################################################################################################################################################################################################### 100.0%
Computing checksum with sha256sum
Checksums matched!
Now using node v18.18.2 (npm v9.8.1)
❯ nvm use 18
Now using node v18.18.2 (npm v9.8.1)
❯ npm sbom
Unknown command: "sbom"

To see a list of supported npm commands, run:
  npm help

```

Reply:
Okay, thanks. Perhaps it would be better to update the node version in the setup step to use Node 20. Either way is fine with me, though.